.github/workflows/feature.yml

```diff
@@ -0,0 +1,27 @@
+name: Feature CI
+
+on: push
+
+jobs:
+  unit-test:
+    name: Unit tests
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Gradle cache
+        uses: actions/cache@v1
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle.kts') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+      - name: Set up JDK 8
+        uses: actions/setup-java@v1
+        with:
```

Comment:
wouldn't it make sense to setup matrix for all LTS versions?

Reply:
As per https://github.com/JUtupe/ktor-rabbitmq/blob/develop/build.gradle#L40-L45 we are currently targetting Java 8. I don't think that we need to test other versions additionally.

Reply:
Does that mean it is necessarily compatible, though? For an example, Spring used to target Java 8 for a long time, but that didn't automatically ensure compatibility with new major Java versions as they were released, those had to be baked in for each release separately. Now this may or may not be the case for `ktor-rabbitmq` (after all, it is significantly simpler than Spring), but having extra confidence at virtually no cost doesn't sound too bad.